src/Components/SmartComponents/CVEs/CVEs.spec.ct.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import CVEs from './CVEs';
+import ReducerRegistry from '../../../Utilities/ReducerRegistry';
+
+import cvesResponse from '../../../../cypress/fixtures/cves.json';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations';
+import messages from '../../../../locales/en.json';
+
+beforeEach(() => {
```

Comment:
make sure that this function (with `mount` call inside) will be executed before each `describe`, but not `it`. if you want to have a fresh new mount for each `it`, then move this function inside `describe`s